tests.js

```diff
@@ -5095,6 +5178,150 @@ window.Specs = {
+					"dev": "#cyrillic-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"lower-belorussian", "lower-bulgarian", "lower-macedonian", "lower-russian", "lower-russian-full",
+					"lower-serbo-croatian", "lower-ukrainian", "lower-ukrainian-full", "upper-belorussian",
+					"upper-bulgarian", "upper-macedonian", "upper-russian", "upper-russian-full", "upper-serbo-croatian",
+					"upper-ukrainian", "upper-ukrainian-full"
+				]
+			},
+			"Devanagari": {
+				"links": {
+					"tr": "#devanagari-styles",
+					"dev": "#devanagari-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"hindi"
+				]
+			},
+			"Ethiopic": {
+				"links": {
+					"tr": "#ethiopic-styles",
+					"dev": "#ethiopic-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"afar", "agaw", "ari", "blin", "dizi", "gedeo", "gumuz", "hadiyya", "harari", "kaffa", "kebena",
+					"kembata", "konso", "kunama", "meen", "oromo", "saho", "sidama", "silti", "tigre", "wolaita", "yemsa"
+				]
+			},
+			"Greek": {
+				"links": {
+					"tr": "#greek-styles",
+					"dev": "#greek-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"greek-lower-modern", "greek-upper-modern", "greek-lower-ancient", "greek-upper-ancient"
+				]
+			},
+			"Han Ideographic": {
+				"links": {
+					"tr": "#chinese-styles",
+					"dev": "#chinese-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"circled-ideograph", "parenthesized-ideograph"
+				]
+			},
+			"Japanese": {
+				"links": {
+					"tr": "#kana-styles",
+					"dev": "#kana-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"circled-katakana"
+				]
+			},
+			"Korean": {
+				"links": {
+					"tr": "#korean-styles",
+					"dev": "#korean-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"circled-korean-consonant", "circled-korean-syllable", "korean-consonant", "korean-syllable",
+					"parenthesized-hangul-consonant", "parenthesized-hangul-syllable"
+				]
+			},
+			"Latin": {
+				"links": {
+					"tr": "#latin-styles",
+					"dev": "#latin-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"fullwidth-lower-alpha", "fullwidth-upper-alpha", "lower-alpha-symbolic", "lower-oromo-qubee",
+					"upper-alpha-symbolic", "upper-oromo-qubee", "parenthesized-lower-latin"
+				]
+			},
+			"Lepcha": {
+				"links": {
+					"tr": "#lepcha-styles",
+					"dev": "#lepcha-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"lepcha"
+				]
+			},
+			"Tamil": {
+				"links": {
+					"tr": "#tamil-styles",
+					"dev": "#tamil-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"ancient-tamil"
+				]
+			},
+			"European Digits & Roman": {
+				"links": {
+					"tr": "#digit-styles",
+					"dev": "#digit-styles",
+					"mdn": "list-style-type"
+				},
+				"tests": [
+					"binary", "circled-decimal", "circled-lower-latin", "circled-upper-latin", "decimal-leading-zero",
```

Comment:
I noticed some duplicate tests here, such as `decimal-leading-zero`, which is already included in CSS Counter Styles Level 3.

Reply:
OK.

Reply:
As mentioned in the other note, this whole WG Note might be removed. If that's done, the duplications are also gone.